README.md

```diff
@@ -100,6 +100,12 @@ You can now install the latest beta version of minishift.
   $ brew cask install minishift-beta
 ```
 
+### Un-installing Minishift
+
+.1 Try to delete the Minishift VM: `minishift delete`
+.1 Remove the folders: `~/.minishift` and `~/.kube`
+.1 Check that there are no remaining Minishift VM. Use KVM (virsh), VirtualBox or Xhyve depending on your driver.
```

Comment:
I do not think we have a way to check if there is an existing VM in Xhyve. So we should add a note about Xhyve that user can not findout if the VM is present in Xhyve.